api/version/2.7.0.md

```diff
@@ -1,6 +1,6 @@
 # ECE 2.7.0 Command Declaration
 
-Total of 359 commands found
+Total of 361 commands found
```

Comment:
The changes in this file should be reverted as we are not making changes to previous versions :)